packages/transformer/src/IModelExporter.ts

```diff
@@ -927,26 +927,27 @@ export class ChangedInstanceIds {
   private elementECClassIds = new Set<string>();
   private aspectECClassIds = new Set<string>();
   private relationshipECClassIds = new Set<string>();
+  private _ecClassIdsInitialized = false;
```

Comment:
personally I would prefer we just have the sets be `undefined` until initialized, and have a getter with the same name just check if they're undefined.

Reply:
changed